src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/InitialConsiderationHelper.java

```diff
@@ -0,0 +1,182 @@
+                        .map(EtIcudlHearing::getEtIcbuCheckComplianceOrders).orElse(null))
+                .hearingNotListedOtherDirections(caseData.getEtICHearingNotListedAnyOtherDirections())
+                //further information
+                .furtherInformation(caseData.getEtICFurtherInformation())
+                .furtherInfoGiveDetails(caseData.getEtICFurtherInformationGiveDetails())
+                .furtherInfoTimeToComply(caseData.getEtICFurtherInformationTimeToComply())
+                .r27ClaimToBe(Optional.ofNullable(caseData.getEtInitialConsiderationRule27())
+                        .map(EtInitialConsiderationRule27::getEtICRule27ClaimToBe).orElse(null))
+                .r27WhichPart(Optional.ofNullable(caseData.getEtInitialConsiderationRule27())
+                        .map(EtInitialConsiderationRule27::getEtICRule27WhichPart).orElse(null))
+                .r27Direction(Optional.ofNullable(caseData.getEtInitialConsiderationRule27())
+                        .map(EtInitialConsiderationRule27::getEtICRule27Direction).orElse(null))
+                .r27NoJurisdictionReason(Optional.ofNullable(caseData.getEtInitialConsiderationRule27())
+                        .map(EtInitialConsiderationRule27::getEtICRule27NoJurisdictionReason).orElse(null))
+                .r27DirectionReason(Optional.ofNullable(caseData.getEtInitialConsiderationRule27())
+                        .map(EtInitialConsiderationRule27::getEtICRule27DirectionReason).orElse(null))
+                .r27NumberOfDays(Optional.ofNullable(caseData.getEtInitialConsiderationRule27())
+                        .map(EtInitialConsiderationRule27::getEtICRule27NumberOfDays).orElse(null))
+                .r28ClaimToBe(Optional.ofNullable(caseData.getEtInitialConsiderationRule28())
+                        .map(EtInitialConsiderationRule28::getEtICRule28ClaimToBe).orElse(null))
+                .r28WhichPart(Optional.ofNullable(caseData.getEtInitialConsiderationRule28())
+                        .map(EtInitialConsiderationRule28::getEtICRule28WhichPart).orElse(null))
+                .r28DirectionReason(Optional.ofNullable(caseData.getEtInitialConsiderationRule28())
+                        .map(EtInitialConsiderationRule28::getEtICRule28DirectionReason).orElse(null))
+                .r28NumberOfDays(Optional.ofNullable(caseData.getEtInitialConsiderationRule28())
+                        .map(EtInitialConsiderationRule28::getEtICRule28NumberOfDays).orElse(null))
+                .furtherInfoAnyOtherDirections(caseData.getEtICFurtherInformationHearingAnyOtherDirections())
+                .build();
+
+        InitialConsiderationDocument document = InitialConsiderationDocument.builder()
+                .accessKey(accessKey)
+                .outputName(IC_OUTPUT_NAME)
+                .templateName(IC_SUMMARY_SC_TEMPLATE_NAME)
+                .data(data).build();
+
+        OBJECT_MAPPER.setSerializationInclusion(JsonInclude.Include.NON_NULL);
```

Comment:
not needed as declared earlier on, may also want to remove the log message further on